### PR TITLE
Release 2.9.1

### PR DESCRIPTION
**Pull request by sergiou87**
## Description

Looking for the PR for the upcoming v2.9.1 production release? Well you've just found it, congratulations! :tada:

This is based on the latest development, which basically is the same as the 2.9.1-beta7 release (tag [release-2.9.1-beta7](https://github.com/desktop/desktop/releases/tag/release-2.9.1-beta7)) and the small fixes from #12817 and #12819.

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
Nothing worth of mentioning
- [x] Verify that all feature flags are flipped appropriately
image
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
No new metrics

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+      "[Fixed] Removed border of Repository and Branch foldouts in dark theme - #12478. Thanks @meJevin!",
+      "[Fixed] Render links in commit messages when they are at the beginning of a line - #12105. Thanks @tsvetilian-ty!",
+      "[Fixed] Show co-authors from undone commits - #12537",
+      "[Fixed] Show SSH prompts (key passphrase, adding host, etc.) to users via dialog - #3457 #8761",
+      "[Improved] Added support for more versions of JetBrains IDEs on Windows - #12778",
+      "[Improved] Check invalidated tokens and prompt the user to sign in again - #12554",
+      "[Improved] Double-click an option when switching branches to quickly confirm how you want changed files to be handled - #12522. Thanks @j-f1!",
+      "[Improved] Improve auth error and prompt to suggest the user use a PAT instead of password - #12323",
+      "[Improved] Increase visibility of misattributed commit warning in dark mode - #12210",
+      "[Improved] Reduced time needed to make a commit - #12529",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Reduce time needed to make a commit - #12529",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+      "[Fixed] Enable 'Open in External Editor' for bat/cmd/sh/exe files - #6361. Thanks @AndreiMaga!",
+      "[Fixed] Green circle in progress dialog remains round when commits have long descriptions - #12594. Thanks @litetex!",
+      "[Fixed] History tab shows all commits as the user scrolls down - #12506 #3704",
+      "[Fixed] Removed border of Repository and Branch foldouts in dark theme - #12478. Thanks @meJevin!",
+      "[Fixed] Render links in commit messages when they are at the beginning of a line - #12105. Thanks @tsvetilian-ty!",
+      "[Fixed] Show co-authors from undone commits - #12537",
+      "[Fixed] Show SSH prompts (key passphrase, adding host, etc.) to users via dialog - #3457 #8761",
+      "[Improved] Added support for more versions of JetBrains IDEs on Windows - #12778",
+      "[Improved] Check invalidated tokens and prompt the user to sign in again - #12554",
+      "[Improved] Double-click an option when switching branches to quickly confirm how you want changed files to be handled - #12522. Thanks @j-f1!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Double-click an option when switching branches to confirm how changed files are handled - #12522. Thanks @j-f1!",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+      "[Added] Show number of lines changed in a commit - #11656",
+      "[Fixed] Change the diff gutter width based on number of lines in diff - #2102. Thanks @ADustyOldMuffin!",
+      "[Fixed] Enable 'Open in External Editor' for bat/cmd/sh/exe files - #6361. Thanks @AndreiMaga!",
+      "[Fixed] Green circle in progress dialog remains round when commits have long descriptions - #12594. Thanks @litetex!",
+      "[Fixed] History tab shows all commits as the user scrolls down - #12506 #3704",
+      "[Fixed] Removed border of Repository and Branch foldouts in dark theme - #12478. Thanks @meJevin!",
+      "[Fixed] Render links in commit messages when they are at the beginning of a line - #12105. Thanks @tsvetilian-ty!",
+      "[Fixed] Show co-authors from undone commits - #12537",
+      "[Fixed] Show SSH prompts (key passphrase, adding host, etc.) to users via dialog - #3457 #8761",
+      "[Improved] Added support for more versions of JetBrains IDEs on Windows - #12778",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Add support for more versions of JetBrains IDEs on Windows - #12778",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
   "releases": {
+    "2.9.1": [
+      "[Added] Add Fluent Terminal shell support - #12305. Thanks @Idered!",
+      "[Added] Add support for IntelliJ CE for macOS - #12748. Thanks @T41US!",
+      "[Added] Show number of lines changed in a commit - #11656",
+      "[Fixed] Change the diff gutter width based on number of lines in diff - #2102. Thanks @ADustyOldMuffin!",
+      "[Fixed] Enable 'Open in External Editor' for bat/cmd/sh/exe files - #6361. Thanks @AndreiMaga!",
+      "[Fixed] Green circle in progress dialog remains round when commits have long descriptions - #12594. Thanks @litetex!",
+      "[Fixed] History tab shows all commits as the user scrolls down - #12506 #3704",
+      "[Fixed] Removed border of Repository and Branch foldouts in dark theme - #12478. Thanks @meJevin!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Remove border of Repository and Branch foldouts in dark theme - #12478. Thanks @meJevin!",
```

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,26 @@
 {
   "releases": {
+    "2.9.1": [
+      "[Added] Add Fluent Terminal shell support - #12305. Thanks @Idered!",
+      "[Added] Add support for IntelliJ CE for macOS - #12748. Thanks @T41US!",
+      "[Added] Show number of lines changed in a commit - #11656",
+      "[Fixed] Change the diff gutter width based on number of lines in diff - #2102. Thanks @ADustyOldMuffin!",
```
Comment (billygriffin) [inline changelog.json]: I think this should also be moved to the improved section.
```suggestion
      "[Improved] Change the diff gutter width based on number of lines in diff - #2102. Thanks @ADustyOldMuffin!",
```

**Review by billygriffin — CHANGES_REQUESTED** [review]
Looks great, with the exception of a couple minor changes!

**Review by EvanMichael09 — CHANGES_REQUESTED** [review]
411. Þ&;$%-,&⁹

**Review by billygriffin — APPROVED** [review]

**Pull request merged**